### DISCUSSION TITLE
Readiness probe failed: HTTP probe failed with statuscode: 409

### DISCUSSION THREAD

**Discussion by kosksq** [tchiotludo/akhq#1305] [Q&A] [answered]
Hi community,

Does anyone have an idea on this issue? I can't properly debug because it shows this error everytime i import my yaml. Especially it happen after i fill in existingSecrets: "". Also, want to ask if there is a solution to type more than one existingSecrets? As i need to import cluster ca cert and user cert.
![image](https://user-images.githubusercontent.com/74179948/169504744-00dddfc6-402e-4c17-a0c6-533f1723e6dc.png)

**Comment by tchiotludo** [comment]
@kosksq don't understand, you have a solution ?
I don't have the time to look at

**Comment by pantelis-karamolegkos** [comment]
I faced the same issue when deploying with `helm`. (although I have `istio` in front and this might complicate things, but I get no errors from `istio`)

**Comment by tchiotludo** [comment, marked as answer]
I think it's related to this : https://github.com/tchiotludo/akhq/commit/5c4b3e6c868b099d7310e12eae277426206ec901

**Comment by pantelis-karamolegkos** [comment]
I think there are more details [here](https://github.com/tchiotludo/akhq/discussions/987).

So my question is how does one pass the `application.yaml` and `application-secrets.yaml` through `values.yaml`?
